cdap-ui/app/features/adapters/routes.js

```diff
@@ -64,7 +70,8 @@ angular.module(PKG.name + '.feature.adapters')
             templateUrl: '/assets/features/adapters/templates/tabs/runs.html',
             controller: 'AdapterRunsController',
             ncyBreadcrumb: {
-              label: 'Runs'
+              parent: 'adapters.list',
+              label: '{{$state.params.adapterId | caskCapitalizeFilter}}'
```

Comment:
Yes. We should probably revert that back. The user is not going to copy paste things from breadcrumb. We should probably have it capitalized.